### ISSUE TITLE
Potential bug in averaging loss

### ISSUE THREAD

**Issue by EgorLakomkin** [SeanNaren/deepspeech.pytorch#317]
Here https://github.com/SeanNaren/deepspeech.pytorch/blob/master/train.py#L289 loss is divided by input.size(0) with a comment that averaging is done by minibatch.
But out has has TxNxH dimensions meaning that loss is averaged by the length. Shouldnt it be loss = loss/out.size(1)?

**Comment by EgorLakomkin** [comment]
Too much work did not spot that out is divided by inputs.size(0) :)
